wp-content/themes/core/components/blocks/quote/Quote_Block_Controller.php

```diff
@@ -0,0 +1,166 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\blocks\quote;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Blocks\Types\Quote\Quote as Quote_Block;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\quote\Quote_Controller;
+use Tribe\Project\Templates\Models\Image;
+use Tribe\Project\Theme\Config\Image_Sizes;
+
+/**
+ * Class Quote
+ */
+class Quote_Block_Controller extends Abstract_Controller {
+
+	public const LAYOUT            = 'layout';
+	public const MEDIA             = 'media';
+	public const QUOTE_TEXT        = 'quote_text';
+	public const CITE_NAME         = 'cite_name';
+	public const CITE_TITLE        = 'cite_title';
+	public const CITE_IMAGE        = 'cite_image';
+	public const CONTAINER_CLASSES = 'container_classes';
+	public const MEDIA_CLASSES     = 'media_classes';
+	public const CONTENT_CLASSES   = 'content_classes';
+	public const CLASSES           = 'classes';
+	public const ATTRS             = 'attrs';
+
+	public string $layout;
+	public ?int $media;
+	public string $cite_name;
+	public string $cite_title;
+	public ?int $cite_image;
+	public string $quote_text;
+	public array $container_classes;
+	public array $media_classes;
+	public array $content_classes;
+	public array $classes;
+	public array $attrs;
+
+	public function __construct( array $args = [] ) {
+		$args                    = $this->parse_args( $args );
+		$this->layout            = $args[ self::LAYOUT ];
+		$this->media             = $args[ self::MEDIA ];
+		$this->cite_name         = $args[ self::CITE_NAME ];
+		$this->cite_title        = $args[ self::CITE_TITLE ];
+		$this->cite_image        = $args[ self::CITE_IMAGE ];
+		$this->quote_text        = $args[ self::QUOTE_TEXT ];
+		$this->container_classes = (array) $args[ self::CONTAINER_CLASSES ];
```

Comment:
Do we still need the `(array)` typecasting for these args?

Reply:
Yes. It ensures that we don't get a type error if the wrong kind of args were passed in.

Reply:
Should we be applying this for all args then?

Reply:
Yes, I think it's a good habit to be in.

Reply:
I've wondered that. Someone could still pass in an different value by error and this would help catch that. But do we want to enable that or throw and error down the road to enforce them setting the correct value? I'm thinking maybe the latter?

Reply:
The robustness principle pushes me to handle it safely in the controller. While I'd like to think that we'd catch the errors in dev/qa, I don't 100% trust our process to catch it every time. I'd rather know that it's save than break the page in production.